ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
```

Comment:
I hereby accept the fact that you're using MacBook, would you kindly move this to global gitignore? ^^

Reply:
Done